xds/src/main/java/io/grpc/xds/ClusterImplLoadBalancer.java

```diff
@@ -362,10 +391,12 @@ public PickResult pickSubchannel(PickSubchannelArgs args) {
             }
           }
           final ClusterLocalityStats stats =
-              result.getSubchannel().getAttributes().get(ATTR_CLUSTER_LOCALITY_STATS);
+              result.getSubchannel().getAttributes().get(ATTR_CLUSTER_LOCALITY).get()
+                  .getClusterLocalityStats();
           if (stats != null) {
```

Comment:
Before this PR this check was handling the case where a subchannel was used that didn't have ATTR_CLUSTER_LOCALITY_STATS set. Can you preserve that check?

Reply:
Added the check along with the null check for stats. PTAL